project/AppDependencies.scala

```diff
@@ -4,6 +4,7 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
+    "org.typelevel"           %% "cats-core"                  % "1.1.0",
```

Comment:
use latest stable version

Reply:
I don't think we actually use this any more

Reply:
Removed